torch/distributed/_sharded_tensor/api.py

```diff
@@ -127,17 +127,22 @@ def _register_remote_shards(sharded_tensor_id: int, rrefs: List[rpc.RRef[Shard]]
 
 class CreateOp(Enum):
     EMPTY = 0
-    ONES = 1
+    FULL = 1
+    ONES = 2
+    RAND = 3
+    ZEROS = 4
 
 
 @dataclass
 class TensorInitParams(object):
     """ Container for list of common params to create new local tensor. """
 
-    __slots__ = ['create_op', 'dtype', 'layout', 'requires_grad', 'pin_memory',
-                 'memory_format']
+    __slots__ = ['create_op', 'fill_value', 'dtype', 'layout', 'requires_grad',
+                 'pin_memory', 'memory_format']
 
     create_op: CreateOp
+    # needed when create_op is FULL
+    fill_value: Number
     dtype: torch.dtype
```

Comment:
Actually it might be useful to have default values for all of these parameters exception  `create_op` so it is simpler to create TensorInitParams.

Reply:
SG, done